ci-operator/config/openshift/runtime-utils/openshift-runtime-utils-master.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: "7"
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+promotion:
+  name: "4.3"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 12Gi
+    requests:
+      cpu: "3"
+      memory: 8Gi
+tag_specification:
+  name: "4.3"
+  namespace: ocp
+tests:
+- as: verify
+  commands: |
+    make verify
+  container:
+    from: src
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: |
+    mkdir -p /tmp/artifacts
+    make test-unit JUNITFILE=/tmp/artifacts/unit_report.xml
+  container:
+    from: bin
```

Comment:
are you even sure this needs to be `bin` and not just `src` ?

Reply:
@bparees fixed.